auth-web/src/components/auth/manage-business/BusinessLookup.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <!-- once in Summary state, need to re-mount to reuse this component -->
-  <div id="business-lookup" v-if="state !== States.SUMMARY">
```

Comment:
Removed this so it wont need a remount, i'll look at this more in the next ticket